packages/web-reporter-ui/CPUReport.tsx

```diff
@@ -69,7 +69,7 @@ export const CPUReport = ({
         }
         maxValue={100}
       />
-      <Accordion>
+      <Accordion TransitionProps={{ unmountOnExit: true }}>
```

Comment:
Having perf problems on perf tools be like:
![image](https://user-images.githubusercontent.com/40902940/213999271-cafdf7e7-cb19-4440-ac90-c554faabf27e.png)